lighthouse-core/report/scripts/lighthouse-report.js

```diff
@@ -47,9 +49,19 @@ LighthouseReport.prototype = {
     }
   },
 
+  toggleHelpText: function(e) {
+    if (e.target.classList.contains('report-section__item-help-toggle')) {
+      const el = e.currentTarget.parentNode.querySelector('.report-section__item-helptext');
+      el.hidden = !el.hidden;
+    }
+  },
+
   addEventListeners: function() {
     this.printButton.addEventListener('click', this.onPrint);
     this.checkboxToggleView.addEventListener('change', this.updateView);
+    Array.from(this.itemDetails).forEach(node => {
```

Comment:
[...this.itemDetails].forEach

should be fine, since itemDetails is an iterable